tests/tests_data/test_algorithms.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
+# Copyright (c) DIRECT Contributors
+
+"""Tests for the direct.algorithms module."""
+
+import pytest
+import torch
+
+from direct.algorithms.optimization import MaximumEigenvaluePowerMethod
+
+
+@pytest.mark.parametrize("size", [20, 30])
+def test_power_method(size):
+    mat = torch.rand((size, size)) + torch.rand((size, size)) * 1j
+    x0 = torch.ones(size) + 0 * 1j
+
+    def A(x):
+        return mat @ x
+
+    algo = MaximumEigenvaluePowerMethod(A, x0)
+    algo()
+
```

Comment:
I would convert this to a `.fit()`, that feels more natural. Maybe even `fit(x0)`?

Reply:
Done